modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/resources/META-INF/resources/js/core/config/initialState.es.js

```diff
@@ -12,11 +12,34 @@
  * details.
  */
 
+const INITIAL_PAGES = [
+	{
+		description: '',
+		localizedDescription: {
+			[themeDisplay.getLanguageId()]: '',
+		},
+		localizedTitle: {
+			[themeDisplay.getLanguageId()]: '',
+		},
+		rows: [
+			{
+				columns: [
+					{
+						fields: [],
+						size: 12,
+					},
+				],
+			},
+		],
+		title: '',
+	},
+];
+
 export const INITIAL_STATE = {
 	activePage: 0,
 	defaultLanguageId: themeDisplay.getLanguageId(),
 	editingLanguageId: themeDisplay.getDefaultLanguageId(),
 	focusedField: {},
-	pages: [],
+	pages: INITIAL_PAGES,
```

Comment:
hey @miltonmc did you check if when creating a new structure in WEM the pages is coming with an empty Array? otherwise, I think this INITIAL here will not work.

Reply:
It does not caming with an empty array

Reply:
Great!!